modules/button/react/stories/stories_visualTesting.tsx

```diff
@@ -364,6 +365,27 @@ const buttonStories = [
       </React.Fragment>
     ),
   },
+  {
+    name: 'Toolbar Dropdown Button',
```

Comment:
Note: This will have to change with #688

Reply:
@anicholls i changed this but I have one question, for the themed visual stories, why is the pattern `Testing|React/Buttons/Button` instead of something like `Testing|React/Buttons/Hyper link/theming` ? 

Also i noticed that for `IconButton` theming this is the path: https://github.com/anicholls/canvas-kit/blob/update-prerelease/modules/button/react/stories/visual-testing/theming/stories_IconButton.tsx#L7

And it doesn't match the other theming paths : https://github.com/anicholls/canvas-kit/blob/update-prerelease/modules/button/react/stories/visual-testing/theming/stories_DeleteButton.tsx#L7 not sure if this is intentional or not

Reply:
@mannycarrera4 the hierarchy is as follows:

`Testing|<TARGET>/<CATEGORY>/<MODULE>/<COMPONENT>`

It's a bit weird to see `Buttons/Button`, but it makes more sense when you see something like `Inputs/TextInput`. The full name of the stories comes from the exported story names, so we end up with `Testing|React/Buttons/Button/Hyperink/Hyperlink States` and `Testing|React/Buttons/Button/Hyperink/Hyperlink Theming States` (note - we repeat the component name in the final story name so that it's easier to read in Chromatic).

Let me know if you still have questions after this explanation.

---

Good catch on the IconButton path. We can update that in #688.

Reply:
awesome thanks for explaining that!  That makes sense